fetch/api/request/destination/fetch-destination.https.html

```diff
@@ -194,6 +194,42 @@
   });
 }, 'HTMLLinkElement with rel=stylesheet fetches with a "style" Request.destination');
 
+// Import declaration with `type: "css"` - style destination
+promise_test(async t => {
+  await new Promise((resolve, reject) => {
```

Comment:
as below, I think you can remove the failureCallback and just return this promise